MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs

```diff
@@ -645,6 +660,35 @@ private static DataFeedEnum<T> GenerateEnumField<T>(IReadOnlyList<string> path,
             return enumField;
         }
 
+        private static IEnumerable<DataFeedItem> GenerateFlagsEnumFields<T>(IReadOnlyList<string> path, IDefiningConfigKey<T> configKey)
+            where T : Enum
+        {
+            var flagsEnumGroup = new DataFeedGroup();
+            flagsEnumGroup.InitBase(configKey.FullId, path, [configKey.Section.Id], configKey.GetLocaleKey("Name").AsLocaleKey());
+            flagsEnumGroup.InitDescription(configKey.GetLocaleKey("Description").AsLocaleKey());
+            yield return flagsEnumGroup;
+
+            var flagsGrouping = new[] { configKey.Section.Id, configKey.FullId };
+
+            foreach (var value in Enum.GetValues(configKey.ValueType).Cast<T>())
+            {
+                var name = value.ToString();
+                var longValue = Convert.ToInt64(value);
```

Comment:
I hate having to do it like this, but that seems to be the only way to do it for a generic enum